caas/kubernetes/provider/constants.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2019 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package provider
+
+const (
```

Comment:
There's consts already in caas/metadata.go

Reply:
no, i didn't find it in metadata.go.
it's all using strings currently